example-config.json

```diff
@@ -30,7 +30,12 @@
         "port":51233
     },
     "log_level":"debug",
-    "log_file":"./clio.log",
+    "log_to_console": true,
+    "log_to_file": true,
+    "log_rotation_size": 2147483648,
+    "log_directory_max_size": 53687091200,
+    "log_rotation_hour_interval": 12,
+    "log_directory":"./clio_log/",
```

Comment:
This will produce this path for the log file by default:
`/var/log/clio/clio.log/clio.log`